package.json

```diff
@@ -117,6 +117,9 @@
   },
   "main": "./dist/es5/index.js",
   "types": "./index.d.ts",
+  "browser": {
```

Comment:
Can't we just use `"browser": "./dist/es5/exceljs.browser.js"`?

Reply:
Sure, it is possible (already changed).